packages/mobile/src/analytics/ValoraAnalytics.ts

```diff
@@ -0,0 +1,150 @@
+import Analytics, { Analytics as analytics } from '@segment/analytics-react-native'
+import Firebase from '@segment/analytics-react-native-firebase'
+import DeviceInfo from 'react-native-device-info'
+import { AppEvents } from 'src/analytics/Events'
+import { AnalyticsPropertiesList } from 'src/analytics/Properties'
+import { SEGMENT_API_KEY } from 'src/config'
+import { store } from 'src/redux/store'
+import Logger from 'src/utils/Logger'
+import * as Web3Utils from 'web3-utils'
+
+const TAG = 'ValoraAnalytics'
+
+async function getDeviceInfo() {
+  return {
+    AppName: DeviceInfo.getApplicationName(),
+    Brand: DeviceInfo.getBrand(),
+    BuildNumber: DeviceInfo.getBuildNumber(),
+    BundleId: DeviceInfo.getBundleId(),
+    Carrier: await DeviceInfo.getCarrier(),
+    DeviceId: DeviceInfo.getDeviceId(),
+    DeviceName: await DeviceInfo.getDeviceName(), // NOTE(nitya) this might contain PII, monitor
```

Comment:
ditto

Reply:
@nityas are you good with cutting this? I agree that the probability of it containing PII is pretty high.

Reply:
+1 to removing this